### ISSUE TITLE
fetchOrderBook bit-z

### ISSUE THREAD

**Issue by Showgofar** [ccxt/ccxt#3157]
- OS: Debian
- Programming Language version: PHP
- CCXT version: latest
- Exchange: bit-z
- Method: fetchOrderBook

Hi could you please reverse asks order in order book
https://www.bit-z.com/api_v1/depth?coin=eth_btc
so it will be in the same order like on other exchanges from best to worst  (best ask will be first)

**Comment by kroitor** [comment]
@Showgofar it should be sorted if you have the most recent version of CCXT. Actually, it's been like that for months. Are you sure that you have the latest version?

```PHP
mbp:ccxt igorkroitor$ php -f examples/php/cli.php bitz fetch_order_book XRB/BTC
./keys.local.json
bitz->fetch_order_book (XRB/BTC)
Array
(
    [bids] => Array
        (
            [0] => Array
                (
                    [0] => 0.00036501
                    [1] => 3.3715
                )

            [1] => Array
                (
                    [0] => 0.000365
                    [1] => 54.3221
                )

            [2] => Array
                (
                    [0] => 0.00033896
                    [1] => 1
                )

            [3] => Array
                (
                    [0] => 0.00024219
                    [1] => 0.1
                )

            [4] => Array
                (
                    [0] => 0.00021797
                    [1] => 0.9999
                )

            [5] => Array
                (
                    [0] => 0.00010797
                    [1] => 0.999
                )

            [6] => Array
                (
                    [0] => 0.0001
                    [1] => 16.6
                )

            [7] => Array
                (
                    [0] => 1.011E-5
                    [1] => 654.3445
                )

            [8] => Array
                (
                    [0] => 1.01E-5
                    [1] => 200
                )

            [9] => Array
                (
                    [0] => 8.9E-6
                    [1] => 30
                )

            [10] => Array
                (
                    [0] => 4.6E-6
                    [1] => 60
                )

            [11] => Array
                (
                    [0] => 9.0E-7
                    [1] => 1000
                )

            [12] => Array
                (
                    [0] => 8.9E-7
                    [1] => 300
                )

            [13] => Array
                (
                    [0] => 4.7E-7
                    [1] => 2000
                )

            [14] => Array
                (
                    [0] => 4.6E-7
                    [1] => 600
                )

            [15] => Array
                (
                    [0] => 1.1E-7
                    [1] => 20000
                )

            [16] => Array
                (
                    [0] => 1.0E-7
                    [1] => 3000
                )

            [17] => Array
                (
                    [0] => 8.0E-8
                    [1] => 3000
                )

            [18] => Array
                (
                    [0] => 5.0E-8
                    [1] => 20000
                )

            [19] => Array
                (
                    [0] => 4.0E-8
                    [1] => 6000
                )

            [20] => Array
                (
                    [0] => 1.0E-8
                    [1] => 20000
                )

        )

    [asks] => Array
        (
            [0] => Array
                (
                    [0] => 0.0042
                    [1] => 5.74
                )

            [1] => Array
                (
                    [0] => 0.00361388
                    [1] => 3.5121
                )

            [2] => Array
                (
                    [0] => 0.0033
                    [1] => 7.7105
                )

            [3] => Array
                (
                    [0] => 0.003
                    [1] => 47.5084
                )

            [4] => Array
                (
                    [0] => 0.0027
                    [1] => 3.0199
                )

            [5] => Array
                (
                    [0] => 0.002599
                    [1] => 1
                )

            [6] => Array
                (
                    [0] => 0.00255
                    [1] => 35.3761
                )

            [7] => Array
                (
                    [0] => 0.0025386
                    [1] => 10.5627
                )

            [8] => Array
                (
                    [0] => 0.0025
                    [1] => 1.557
                )

            [9] => Array
                (
                    [0] => 0.00249
                    [1] => 116
                )

            [10] => Array
                (
                    [0] => 0.002449
                    [1] => 33.33
                )

            [11] => Array
                (
                    [0] => 0.002399
                    [1] => 1
                )

            [12] => Array
                (
                    [0] => 0.002345
                    [1] => 33.33
                )

            [13] => Array
                (
                    [0] => 0.0023445
                    [1] => 40.6999
                )

            [14] => Array
                (
                    [0] => 0.00234
                    [1] => 12.5333
                )

            [15] => Array
                (
                    [0] => 0.002325
                    [1] => 10.5501
                )

            [16] => Array
                (
                    [0] => 0.0023
                    [1] => 50
                )

            [17] => Array
                (
                    [0] => 0.00228
                    [1] => 2.3013
                )

            [18] => Array
                (
                    [0] => 0.00203302
                    [1] => 7.3464
                )

            [19] => Array
                (
                    [0] => 0.00202291
                    [1] => 5.7757
                )

            [20] => Array
                (
                    [0] => 0.00201285
                    [1] => 7.7375
                )

            [21] => Array
                (
                    [0] => 0.00200867
                    [1] => 0.945
                )

            [22] => Array
                (
                    [0] => 0.00200284
                    [1] => 10.783
                )

            [23] => Array
                (
                    [0] => 0.002
                    [1] => 101.3937
                )

            [24] => Array
                (
                    [0] => 0.001999
                    [1] => 9.4504
                )

            [25] => Array
                (
                    [0] => 0.00199288
                    [1] => 1.8463
                )

            [26] => Array
                (
                    [0] => 0.00198297
                    [1] => 3.8486
                )

            [27] => Array
                (
                    [0] => 0.00197586
                    [1] => 5
                )

            [28] => Array
                (
                    [0] => 0.00197311
                    [1] => 4.2369
                )

            [29] => Array
                (
                    [0] => 0.0019633
                    [1] => 12.251
                )

            [30] => Array
                (
                    [0] => 0.00195354
                    [1] => 8.5158
                )

            [31] => Array
                (
                    [0] => 0.00194383
                    [1] => 5.4702
                )

            [32] => Array
                (
                    [0] => 0.00193416
                    [1] => 5.3691
                )

            [33] => Array
                (
                    [0] => 0.00192454
                    [1] => 1.7356
                )

            [34] => Array
                (
                    [0] => 0.00191497
                    [1] => 9.5617
                )

            [35] => Array
                (
                    [0] => 0.00190545
                    [1] => 6.7884
                )

            [36] => Array
                (
                    [0] => 0.00189598
                    [1] => 8.9386
                )

            [37] => Array
                (
                    [0] => 0.00188655
                    [1] => 9.5421
                )

            [38] => Array
                (
                    [0] => 0.00187717
                    [1] => 5.8473
                )

            [39] => Array
                (
                    [0] => 0.00186784
                    [1] => 11.2109
                )

            [40] => Array
                (
                    [0] => 0.00185855
                    [1] => 8.8017
                )

            [41] => Array
                (
                    [0] => 0.00184931
                    [1] => 10.4289
                )

            [42] => Array
                (
                    [0] => 0.00184011
                    [1] => 7.5946
                )

            [43] => Array
                (
                    [0] => 0.00183925
                    [1] => 96.87
                )

            [44] => Array
                (
                    [0] => 0.00183096
                    [1] => 5.1718
                )

            [45] => Array
                (
                    [0] => 0.00182186
                    [1] => 4.5657
                )

            [46] => Array
                (
                    [0] => 0.0018128
                    [1] => 8.0398
                )

            [47] => Array
                (
                    [0] => 0.00181
                    [1] => 10.01
                )

            [48] => Array
                (
                    [0] => 0.00180379
                    [1] => 7.6621
                )

            [49] => Array
                (
                    [0] => 0.00179482
                    [1] => 11.4083
                )

            [50] => Array
                (
                    [0] => 0.0017859
                    [1] => 1.9522
                )

            [51] => Array
                (
                    [0] => 0.00177702
                    [1] => 7.3575
                )

            [52] => Array
                (
                    [0] => 0.00176818
                    [1] => 9.6403
                )

            [53] => Array
                (
                    [0] => 0.00175999
                    [1] => 0.7177
                )

            [54] => Array
                (
                    [0] => 0.00175939
                    [1] => 2.1196
                )

            [55] => Array
                (
                    [0] => 0.00175064
                    [1] => 9.8421
                )

            [56] => Array
                (
                    [0] => 0.00174194
                    [1] => 7.3054
                )

            [57] => Array
                (
                    [0] => 0.00173328
                    [1] => 9.0073
                )

            [58] => Array
                (
                    [0] => 0.00172466
                    [1] => 7.1574
                )

            [59] => Array
                (
                    [0] => 0.0017
                    [1] => 10
                )

            [60] => Array
                (
                    [0] => 0.0016
                    [1] => 10
                )

            [61] => Array
                (
                    [0] => 0.00157999
                    [1] => 0.7641
                )

            [62] => Array
                (
                    [0] => 0.00157033
                    [1] => 10.1754
                )

            [63] => Array
                (
                    [0] => 0.0015337
                    [1] => 10.137
                )

            [64] => Array
                (
                    [0] => 0.00152008
                    [1] => 2.8769
                )

            [65] => Array
                (
                    [0] => 0.00151056
                    [1] => 4.4917
                )

            [66] => Array
                (
                    [0] => 0.00151021
                    [1] => 0.616
                )

            [67] => Array
                (
                    [0] => 0.0015
                    [1] => 10
                )

            [68] => Array
                (
                    [0] => 0.00145
                    [1] => 10.01
                )

            [69] => Array
                (
                    [0] => 0.00143
                    [1] => 5.3946
                )

            [70] => Array
                (
                    [0] => 0.0014
                    [1] => 25.3184
                )

            [71] => Array
                (
                    [0] => 0.00135999
                    [1] => 2.3341
                )

            [72] => Array
                (
                    [0] => 0.00131
                    [1] => 23.8361
                )

            [73] => Array
                (
                    [0] => 0.0013
                    [1] => 69.0374
                )

            [74] => Array
                (
                    [0] => 0.0011
                    [1] => 10
                )

            [75] => Array
                (
                    [0] => 0.00108
                    [1] => 24.5379
                )

            [76] => Array
                (
                    [0] => 0.001058
                    [1] => 9.3052
                )

            [77] => Array
                (
                    [0] => 0.00105308
                    [1] => 119.3241
                )

            [78] => Array
                (
                    [0] => 0.001053
                    [1] => 7.2946
                )

            [79] => Array
                (
                    [0] => 0.001052
                    [1] => 0.7613
                )

            [80] => Array
                (
                    [0] => 0.001051
                    [1] => 0.9762
                )

            [81] => Array
                (
                    [0] => 0.00105
                    [1] => 10.01
                )

            [82] => Array
                (
                    [0] => 0.001
                    [1] => 3.9239
                )

            [83] => Array
                (
                    [0] => 0.000994
                    [1] => 8.1918
                )

            [84] => Array
                (
                    [0] => 0.00098118
                    [1] => 6.31
                )

            [85] => Array
                (
                    [0] => 0.00089
                    [1] => 2.8946
                )

            [86] => Array
                (
                    [0] => 0.00088997
                    [1] => 5
                )

            [87] => Array
                (
                    [0] => 0.00082797
                    [1] => 1.6778
                )

            [88] => Array
                (
                    [0] => 0.00078118
                    [1] => 0.5987
                )

            [89] => Array
                (
                    [0] => 0.000578
                    [1] => 186.6836
                )

            [90] => Array
                (
                    [0] => 0.00057
                    [1] => 3.8135
                )

            [91] => Array
                (
                    [0] => 0.00053584
                    [1] => 10.0625
                )

            [92] => Array
                (
                    [0] => 0.00053583
                    [1] => 7.8896
                )

            [93] => Array
                (
                    [0] => 0.00053281
                    [1] => 9.1817
                )

            [94] => Array
                (
                    [0] => 0.0005328
                    [1] => 17.4287
                )

            [95] => Array
                (
                    [0] => 0.00045
                    [1] => 159.506
                )

            [96] => Array
                (
                    [0] => 0.00044788
                    [1] => 182.9852
                )

            [97] => Array
                (
                    [0] => 0.0004469
                    [1] => 26.8367
                )

            [98] => Array
                (
                    [0] => 0.00044667
                    [1] => 36.573
                )

            [99] => Array
                (
                    [0] => 0.00044666
                    [1] => 9.0698
                )

        )

    [timestamp] => 1529246398000
    [datetime] => 2018-06-17T14:39:58.000Z
    [nonce] =>
)
mbp:ccxt igorkroitor$
```

**Comment by Showgofar** [comment]
Hi. yes im sure 
Even in your dump - the best ask is the latest one

**Comment by kroitor** [comment]
Fixed in 1.14.210! Thx for reporting!
